src/language-id-client.ts

```diff
@@ -0,0 +1,107 @@
+import * as FormData from 'form-data';
+import { Readable } from 'stream';
+
+import { BaseApiClient } from './base-api-client';
+import { GetListOfJobsOptions } from './models/GetListOfJobsOptions';
+import { LanguageIdJob } from './models/language-id/LanguageIdJob';
+import { LanguageIdJobOptions } from './models/language-id/LanguageIdJobOptions';
+import { LanguageIdResult } from './models/language-id/LanguageIdResult';
+
+const LanguageIdContentType = 'application/vnd.rev.languageid.v1.0+json';
+
+const TWO_GIGABYTES = 2e9; // Number of Bytes in 2 Gigabytes
+
+/**
+ * Client which handles connection to the Rev AI language id API.
+ */
+export class LanguageIdClient extends BaseApiClient<LanguageIdJob, LanguageIdResult> {
+    /**
+     * @param accessToken Access token used to validate API requests
+     * @param version (optional) version of the API to be used
+     */
+     constructor (accessToken: string) {
```

Comment:
I see we are adding space after ctor word in other files. not sure was a mistake from the beginning. should that space be there?

Reply:
oh yeah I'm not sure there should be, I copied the format from other clients, I can remove space